README.md

```
@@ -1,46 +1,431 @@
+   This will automatically run `prisma generate` to set up the Prisma client.
+
+3. **Set up environment variables**
+   
+   Create a `.env.local` file in the root directory:
+   ```env
+   DATABASE_URL="your_mongodb_connection_string"
+   NEXT_PUBLIC_SITE_URL="http://localhost:3000"
+   ```
+
+   Replace `your_mongodb_connection_string` with your MongoDB connection string. For local development, you can use:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)